frontend/src/Commons/Button.tsx

```diff
@@ -1,7 +1,17 @@
 import StyledButton from './Styles/Button.style';
 
-const Button = (
-  { text }: { text: string }, // Destructured primitive string literal
-) => <StyledButton type="button">{text}</StyledButton>;
+export type ButtonSizes = 'sm' | 'md' | 'lg';
+export type ButtonProps = {
+  handleClick?: () => void;
+  text: string;
+  size: ButtonSizes;
+  disabled: boolean;
+};
```

Comment:
Adds to modularity, cool improvement :rocket:

Reply:
yea 👀